content/en/agent/logs/proxy.md

```diff
@@ -15,8 +15,14 @@ further_reading:
 
 Log collection requires the Datadog Agent v6.0+. Older versions of the Agent do not include the `log collection` interface.
 
+## HTTPS log Forwarding
+
+**We recommend to [configure the Agent to send logs in HTTPS][2]** and to use the same [set of proxy settings][2] as other data types.
```

Comment:
```suggestion
**As a best practice, [configure the Agent to send logs in HTTPS][2]** and to use the same [set of proxy settings][2] as other data types.
```

Reply:
It is actually not a best practice but an actual recommendation. Not sure how we want to phrase it (working on having a graphic to better explain why we recommend it).

In a close future, HTTPS will be the default but we cannot document what is not available today unfortunately.

Reply:
That makes sense, thanks for the info. Updated to _it is recommended_ as we can't use we.